docker-compose.yml

```diff
@@ -34,26 +40,37 @@ services:
       - fpm
     volumes:
       - ./backend/vhost-docker.conf:/etc/nginx/conf.d/laravel.conf
+  laravel_queue_worker:
+    env_file: ./backend/.env.docker
+    build: ./tools/php-queue-worker
+    volumes:
+      - ./backend/:/var/www/laravel
+    links:
+      - db 
+      - redis
+      - ethereum_proxy 
+  laravel_cron_worker:
+    env_file: ./backend/.env.docker
+    build: ./tools/php-cron-worker
+    volumes:
+      - ./backend/:/var/www/laravel
+    links:
+      - db 
+      - redis
+      - ethereum_proxy 
 
   fpm:  
      build: ./backend
      ports:
          - "9000:9000"
      links: 
          - db
+         - redis
          - ethereum_proxy
      volumes:
          - ./backend/:/var/www/laravel
      working_dir: /var/www/laravel
-     environment:
-      DB_HOST: db
-      DB_PORT: 3306
-      DB_CONNECTION: mysql
-      DB_DATABASE: cryptocards
-      DB_USERNAME: root
-      DB_PASSWORD: root
-      RPC_SERVER_ADDRESS: ethereum_proxy:50051
-
+     env_file: ./backend/.env.docker
```

Comment:
we should add an env file for the frontend as well, so that it is easy to switch from localhost to another ip.